RunJavaUtils.py

```diff
@@ -61,7 +62,7 @@ def Compare_Lines(goldenList, studentList):
 # that is the package name.
 # same with the className  
 def Get_Java_Info(fileName):
-    javaFile = open(fileName, "r", encoding="latin-1" )
+    javaFile = io.open(fileName, "r", encoding="latin-1" )
```

Comment:
Out of curiosity, why do we need something other than `open(fileName, 'rt')` here?

Reply:
This handled files with utf encoding, I was getting errors otherwise.